args_test.go

```diff
@@ -135,11 +155,31 @@ var (
 				Shell:     "fish",
 			},
 		},
+		{
+			Args:    []string{"env", "--help"},
+			Command: &Help{Subcommand: "env"},
+		},
 
 		// Help
```

Comment:
You appear to be missing test cases for some of the subcommands: `env`, `ls`, `list`, `load`, `rm`, `shell`, and `upgrade`.